tools/vf-component-library/src/site/patterns/index.njk

```diff
@@ -2,11 +2,10 @@
 is_index: true
 title: Patterns and boilerplates
 subtitle: More than components, this is how you should do things.
+intro: Where components offer technical implementations, patterns and boilerplates show how things should come together to facilitate tasks and goals.
```

Comment:
I'm leaning towards 'content' being in the front matter being a bit of an anti pattern when we have `.yml` available for it all - probably not the case here -- but in some places it has made using `{% include component %}` 'icky' because you're having to `{% set content as thing %}` in the markup more. 

Anyways - just a drive–by comment for the Monday afternoon